frontend-react/src/content/resources/ResourcesIndex.mdx

```diff
@@ -48,6 +49,24 @@ import { Card, CardHeader, CardBody, CardGroup } from "@trussworks/react-uswds";
             </CardBody>
         </Card>
 
+        <Card gridLayout={{tablet: {col: 4}}}>
+            <CardHeader>
+                ### <a className="usa-link" href="/resources/manage-public-key">Manage your public key <Icon.Lock /></a>
```

Comment:
Can you remove the extra space after "key"? it might be creating that extra underline in the screenshot

Reply:
To clarify: we should still maintain the padding between the text and the icon, but it shouldn't be treated as a space in the URL

Reply:
Total nitpick, but is there anything we can do to vertically center-align the icon to the text?

Reply:
Fixed: 
<img width="333" alt="Screenshot 2023-04-26 at 11 17 30 AM" src="https://user-images.githubusercontent.com/516877/234666956-1f009fcf-0528-4700-865f-f9cac118702d.png">